CONTRIBUTING.md

```diff
@@ -6,14 +6,12 @@ To contribute to this project please carefully read this document.
 
 `wakatime-cli` is written in [Go](https://golang.org/).
 
-**Note:** wakatime-cli uses [Go Modules](https://github.com/golang/go/wiki/Modules) to manage dependencies.
-
 Prerequisites:
 
-- `make` - We use make to build and run tests.
+- We use make to build and run tests.
 - [Go 1.16](https://golang.org/doc/install)
 
-After cloning it first run `make test` to make sure everything is properly set.
+After cloning, install dependencies with `make install`.
```

Comment:
What about:

```
After cloning, install dependencies with `make install` and then run `make test` to make sure everything is properly set.
```

Reply:
The `make test` is already stated below in the testing section... I don't think it needs to be in setup.